Atomic/containers.py

```diff
@@ -120,7 +119,7 @@ def _walk(_filter_objs, _filter, _value):
 
         if not self.args.filter:
             return con_objs
-        filtered_objs = copy.deepcopy(con_objs)
+        filtered_objs = con_objs[:]
```

Comment:
I think (testing my python knowledge here) is changing from a deep copy to a shallow copy.  The deep will make a copy of the original object, the shallow adds a pointer to the original object.  I think that's ok, but just wanted to bring this up on the off chance that you've not mulled it over.